qemu/tests/cfg/rng_bat.cfg

```diff
@@ -33,6 +33,8 @@
             check_rngd_service = "service rngd status"
             start_rngd_service = "service rngd start"
         s390x:
+            !RHEL.7,RHEL.8:
+                update_driver = "echo 'virtio_rng.0' > /sys/devices/virtual/misc/hw_random/rng_current"
             RHEL.9:
                 update_driver = "echo 'virtio_rng.0' > /sys/devices/virtual/misc/hw_random/rng_current"
```

Comment:
Can merge them.

Reply:
@BohdanMar Yihyu means, you should remove the 'RHEL.9:' part cause ! RHEL.7,RHEL.8 means except them equals ...RHEL6,REHL9,RHEL10...

Reply:
Thanks for the explanation. 
Done.
Please check updated version.